docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        jnp.array([[-1.6, -1.6]]),
</pre><p>maybe easier to have only one <code>jnp.array</code> on top of matrix, rather than one per line? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Done